salt/metalk8s/kubernetes/etcd/healthy.sls

```diff
@@ -1,6 +1,13 @@
 include:
   - .installed
 
+Delay after etcd pod deployment:
```

Comment:
I'm wondering, could we maybe add something in `metalk8s.static_pod_managed` that waits for `kubelet` to pick up the change instead of sporadically adding random delays?

Reply:
It's more containerd but yes we could do that good idea. Don't really know how we can do that but I think it's doable. I will create a ticket for this but let merge this for the moment

Reply:
> It's more containerd

Hum, no I think `kubelet` is the one watching `/etc/kubernetes/manifests` and asking to `containerd` for a restart.

> I will create a ticket for this but let merge this for the moment

:+1:

Reply:
> Hum, no I think `kubelet` is the one watching `/etc/kubernetes/manifests` and asking to `containerd` for a restart.

Oh yes indeed it's kubelet watching the manifest and containerd doing the start/stop true